lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -199,8 +199,8 @@ function findDifference(path, actual, expected) {
 
 /**
  * Collate results into comparisons of actual and expected scores on each audit.
- * @param {{finalUrl: string, audits: !Array}} actual
- * @param {{finalUrl: string, audits: !Array}} expected
+ * @param {{finalUrl: string, audits: !Array, errorCode: string}} actual
```

Comment:
```suggestion
 * @param {{finalUrl: string, audits: Array, errorCode?: string}} actual
```

Reply:
I don't think these files are actually type checked though, audits isn't an `Array`...

Reply:
Hah, it certainly isn't. Opened #6852.